### ISSUE TITLE
[Enhancement] Add option to copy Saves from SD card to vWii system memory and also

### ISSUE THREAD

**Issue by DraconicNEO** [Xpl0itU/savemii#37]
Since some saves may be copy protected and vWii doesn't currently have a way to patch nocopy protection (Wii has Priiloader patches but nocopy doesn't work on vWii yet) I think it would be a good idea to add the ability to copy saves from the SD card to the vWii system.

**Comment by Xpl0itU** [comment]
You can already manage all vWii saves, I don’t get the suggestion

**Comment by DraconicNEO** [comment]
> You can already manage all vWii saves, I don’t get the suggestion

I don't think this one lets you access the ones copied to the SD card (the ones copied to the SD card from System menu).
![Screenshot (14)](https://user-images.githubusercontent.com/44508470/224039530-0d2e0ba0-71a7-442d-91ae-448db06b0be4.png)

Would be useful to be able to backup saves that were copied this way, as well as being able to copy them back to system menu since system menu restricts copying protected saves.

**Comment by Xpl0itU** [comment]
Afaik backups made with the system menu are in a different format than the raw saves savemii manages, so I doubt I'll do this, correct me if I'm wrong

**Comment by DraconicNEO** [comment]
> Afaik backups made with the system menu are in a different format than the raw saves savemii manages, so I doubt I'll do this, correct me if I'm wrong

Oh I thought they were stored in much of the same way and that this app and that they were encoded in the same way inside the System memory as on the SD card and that this and other Save Backup tools decrypted/extracted them.

**Comment by TheMasterCoders** [comment]
i think this can be closed due to priiloader on vwii now

**Comment by DraconicNEO** [comment]
> i think this can be closed due to priiloader on vwii now

Agreed, I was under the impression when I started this that Priiloader would be so hard or in such low demand on the Wii U that it just wouldn't happen in any meaningful way, hence why I was suggesting an alternative to the nocopy removal patch. It's good to see that I was wrong and Priiloader did eventually catch up. I'll close this issue now as it has outlived it's usefulness (unless someone thinks copying saves from SD would be a good feature to add, then feel free to reopen this issue).

**Comment by w3irDv** [comment]
Hi! Just letting you know. It's been three years... and may be is of little use now, but I've added this feature in my mod [latest version](https://github.com/w3irDv/savemii/releases/tag/1.8.1)

**Comment by DraconicNEO** [comment]
> Hi! Just letting you know. It's been three years... and may be is of little use now, but I've added this feature in my mod [latest version](https://github.com/w3irDv/savemii/releases/tag/1.8.1)

Very cool.
